src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -544,11 +543,14 @@ public async void UpdateReviewBackground()
                     _telemetryClient.StopOperation(operation);
                 }
             }
+
+            await SyncPackageDisplayServiceName();
```

Comment:
Why does changing the ordering around affect how long this takes?

Reply:
Also, it seems like this is already not awaited from ReviewBackgroundService so I'm curious how it is causing a timeout.

Reply:
We return this background task that runs when system starts and server startup is delayed when there are too many reviews to get service name synced.